### PR TITLE
Rename bitcoin bash completion files so that they refer to zcash and not bitcoin.

### LINKED ISSUE #2167: Rename contrib/bitcoin*.bash-completion
Rename these files to ``contrib/zcash*.bash-completion``.

### PR DESCRIPTION
**Pull request by syd0**
This closes #2167

### CODE REVIEW

**Comment by ebfull** [comment]
utACK. Would love @str4d's input. Thanks @syd0.

File: zcutil/build-debian-package.sh [hunk 1]
```diff
@@ -51,8 +51,8 @@ cp $SRC_DOC/man/zcashd.1 $DEB_MAN
 cp $SRC_DOC/man/zcash-cli.1 $DEB_MAN
 cp $SRC_DOC/man/zcash-fetch-params.1 $DEB_MAN
 # Copy bash completion files
-cp $SRC_PATH/contrib/bitcoind.bash-completion $DEB_CMP/zcashd
-cp $SRC_PATH/contrib/bitcoin-cli.bash-completion $DEB_CMP/zcash-cli
+cp $SRC_PATH/contrib/zcash.bash-completion $DEB_CMP/zcashd
```
Comment (str4d) [inline zcutil/build-debian-package.sh]: `zcashd`

**Comment by syd0** [comment]
Thanks for the catch @str4d! Btw i noticed that zcash-tx isn't included in build-debian-package.sh. Do we want that in there?

**Comment by str4d** [comment]
No, it was left out because it has never actually been a supported released binary. We would want to add shielded transaction support to it before then.

**Comment by syd0** [comment]
@str4d - yep, squashed! Is it preferred in general to have a single commit per PR? Just so I know what to do going forward.

**Comment by daira** [comment]
Had the completion been smoke-tested after these changes?

**Comment by syd0** [comment]
@daira - yep it seems to work! I'm seeing some inconsistencies, eg it doesn't always work depending on whether i do `./src/zcash-cli` vs `src/zcash-cli` vs `cd src ; ./zcash-cli`, but I suspect that's not related to my change.

**Comment by str4d** [comment]
@syd0 wrote:
> Is it preferred in general to have a single commit per PR? Just so I know what to do going forward.

No, multi-commit PRs are fine. I just tend to prefer that "fixup" commits addressing PR review comments are merged into the appropriate commit (possibly updating the commit's message, if it's something we don't want to forget).

**Comment by str4d** [comment]
@zkbot r+

**Pull request merged**